src/components/Popups/ReplyField/PopupList.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { Modifier as PopperModifier, Options } from '@popperjs/core';
+import classnames from 'classnames';
+import PopupBase from '../PopupBase';
+
+import './PopupList.scss';
+
+export type Modifier = PopperModifier<string, Partial<Options>>;
+
+export type Props = {
+    className?: string;
+    reference: HTMLElement;
+    modifiers: [Modifier];
+};
+
+const PopupList = ({ className, reference, modifiers, ...rest }: Props): JSX.Element => {
+    const options = {
+        modifiers: [
+            {
+                name: 'offset',
+                options: {
+                    offset: [0, 3],
+                },
+            },
+            {
+                name: 'eventListeners',
+                options: {
+                    scroll: true,
+                    resize: true,
+                },
+            },
+            ...modifiers,
+        ],
+        placement: 'bottom-start',
+    } as Partial<Options>;
+
+    return (
+        <div className={classnames('ba-PopupList', className)}>
+            <PopupBase options={options} reference={reference} {...rest}>
```

Comment:
PopupBase should be the root component for each popup. You should be able to add class names to it as needed.